binary.cabal

```diff
@@ -1,3 +1,5 @@
+cabal-version:  3.0
```

Comment:
:)

A smaller bump would been sufficient, but OTOH it might be nice to force users to upgrade their `cabal-install` if they ever want to reinstall `binary`!